test/integration/scheduler/queue_test.go

```diff
@@ -311,6 +311,24 @@ func TestCoreResourceEnqueue(t *testing.T) {
 			},
 			wantRequeuedPods: sets.New("pod1"),
 		},
+		{
+			name:         "Pod rejected by the NodeResourcesFit plugin is requeued when a Node is updated",
```

Comment:
Let's add a second pod to this test case, that won't be requeued after the node is updated, e.g. still having too high requests.

Reply:
And could you also cover node add and pod delete events in a yet another test cases?

Reply:
Thank you for reviewing.
I added a second pod that is not requeued.
I also added two other test cases to cover Node add and Pod delete events.
Could you check this, please?